packages/react-celo/src/types.ts

```diff
@@ -60,9 +60,15 @@ export interface WalletConnectProvider extends Provider {
 export interface Connector {
   kit: MiniContractKit;
   type: WalletTypes;
+  /**
+   * Name of the account.
```

Comment:
This an address not name

Reply:
I just moved the comment for that one, but yeah I should fix it.